renormalizer/lib/krylov/krylov.py

```diff
@@ -34,6 +34,8 @@ def expm_krylov(Afunc, dt, vstart: xp.ndarray, block_size=50):
         On Krylov subspace approximations to the matrix exponential operator
         SIAM J. Numer. Anal. 34, 1911 (1997)
     """
+    if not np.iscomplex(dt):
+        dt = dt.real
```

Comment:
If `dt` is of type `complex`, it will turn everything it is operated with to type `complex`, like `dt * np.random.rand(5)`. This is not desired if `dt` is actually a real number with type `complex`.